### PR TITLE
Update SourceMap docs.

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,18 @@ In order to enable source map support you have to specify path to the source map
 
 *Please note* that Airbrake backend downloads source map file in order to process backtrace. This means that source map should be publicly accessible via HTTP. So, for example, don't expect source map support to work on your local webserver running on `localhost`.
 
+You can also specify source map URL in the notice context using `sourceMaps` map, where key is shell file name pattern and value is source map URL:
```
Comment (kyrylo) [inline README.md]: ``using `sourceMaps` map`` looks a bit confusing in JS. It's technically an object, not a map.

I suggest:

> You can also specify a source map URL in the notice context using the special `sourceMaps` object

File: README.md [hunk 2]
```diff
@@ -139,6 +139,18 @@ In order to enable source map support you have to specify path to the source map
...
+
+```js
+airbrake.addFilter(function(notice) {
+  notice.context.sourceMaps = {
+    '*': 'https://domain.com/path/to/source.map', // for all files
+    'https://domain.com/path/to/file.min.js': 'https://domain.com/path/to/source.map'
+  }
```
Comment (kyrylo) [inline README.md]: Missing comma.

Comment (vmihailenco) [inline README.md]: AFAIK comma is not required here.

Comment (kyrylo) [inline README.md]: Sorry, not comma, but semicolon.

File: README.md (hunk 1 above)
Comment (vmihailenco) [inline README.md]: If I say object then the second part about key/value does not make much sense.

File: README.md (hunk 2 above)
Comment (vmihailenco) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (kyrylo) [inline README.md]: I guess `key` can be called `property` (which is a very common name in JS).

Comment (vmihailenco) [inline README.md]: Yes, but I want to say that those are map keys and you can have multiple keys.

Comment (kyrylo) [inline README.md]: What do you think:

> Custom source map URLs are supported by assigning a special property of `notice.context` called `sourceMaps`. The keys of the `sourceMaps` object represent shell filename pattern and the values are links to your source maps.

Comment (vmihailenco) [inline README.md]: Done

**Pull request merged**